researchstack-sdk/src/main/java/org/sagebionetworks/bridge/researchstack/BridgeDataProvider.java

```diff
@@ -484,11 +489,16 @@ public void setUserSharingScope(Context context, String scope) {
 
     @Override
     public SchedulesAndTasksModel loadTasksAndSchedules(Context context) {
-        // TODO: integrate with bridge
-        // forConsentedUsersApi.getSchedules();
-        // forConsentedUsersApi.getScheduledActivities();
+        logger.info("loadTasksAndSchedules()");
+
+        // TODO: figure out the correct arguments to pass here
+        ScheduledActivityList scheduledActivityList = bridgeManagerProvider.getActivityManager().getActivities("-07:00", 1, 0)
```

Comment:
Looks like the argument in iOS is a string version of today's date offset formatted as ISO8601 which is like '+00:00'. 

However, iOS doesn't have that as a parameter to pass in at all so I would just compute it before making the call.